apartment.rb

```diff
@@ -1,2 +1,29 @@
+require_relative 'square_footage'
+
 class Apartment
+  include SquareFootage
+  attr_reader :rooms, :monthly_rent, :number
+
+
+  def initialize(args = {})
+    @rooms = args.fetch(:rooms, [])
+    @number = args.fetch(:number, nil)
+    @monthly_rent = args.fetch(:monthly_rent, 1000)
+  end
+
+  def collection_of_rooms
+    self.rooms
```

Comment:
This is actually a repetitive method. Your attr_reader :rooms creates this getter method for you. Without this code, you could still call .rooms on an instance of the Apartment class. @perohrstrom